pkg/controller/gittrack/gittrack_controller_test.go

```diff
@@ -671,12 +685,12 @@ var _ = Describe("GitTrack Suite", func() {
 
 				var hist prometheus.Histogram
 				Eventually(func() error {
-					var err error
-					hist, err = metrics.TimeToDeploy.GetMetricWith(map[string]string{
+					observer, err := metrics.TimeToDeploy.GetMetricWith(map[string]string{
 						"name":       instance.GetName(),
 						"namespace":  instance.GetNamespace(),
 						"repository": instance.Spec.Repository,
 					})
+					hist = observer.(prometheus.Histogram)
```

Comment:
What's up with these changes? They don't seem related?

Reply:
Result of a dirty `vendor` (from testing a newer version of `prometheus/client_golang`), have removed these changes and rebased